smoke-tests/spec/error_page_spec.rb

```diff
@@ -12,11 +12,142 @@
       }.to raise_error(OpenURI::HTTPError, "404 Not Found")
     end
 
-    xit "serves a 404 response" do
-      URI.open(url)
-    rescue OpenURI::HTTPError => e
-      body = e.io.string
-      expect(body).to eq("404 page not found\n")
+    it "serves a 404 response" do
+      begin
```

Comment:
I'd do this the same way as on lines 51 - 59, just for the sake of keeping things consistent.